exec/java-exec/pom.xml

```diff
@@ -828,31 +828,9 @@
           </execution>
         </executions>
       </plugin>
-      <plugin> <!-- classpath scanning  -->
+      <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>exec-maven-plugin</artifactId>
-        <version>1.2.1</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.drill</groupId>
-            <artifactId>drill-common</artifactId>
```

Comment:
We don't need dependency onto `drill-common` or it is specified in some other place?

Reply:
`drill-common` and `drill-common-test` are included as part of `test` scope dependencies